universal-login-react/src/ui/UFlow/Devices/NewDeviceMessage.tsx

```diff
@@ -14,6 +15,19 @@ export const NewDeviceMessage = ({deployedWallet, onManageClick, className}: New
   const [notifications, setNotifications] = useState([] as Notification[]);
   useEffect(() => deployedWallet.subscribeAuthorisations(setNotifications), []);
 
+  const [isLoading, setIsLoading] = useState(false);
+
+  async function onDeny() {
+    setIsLoading(true);
+    await deployedWallet.denyRequests();
```

Comment:
Shall we wrap it in a try-catch, and `setIsLoading(false)` in `catch` block?

If the request to deny the requests fails, then it would be loading forever. With `setIsLoading(false)` in `catch`, it would be possible to try clicking 'deny' again.